apps/private-law/prl-cron-hearing-actual-task/demo.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: helm.toolkit.fluxcd.io/v2
+kind: HelmRelease
+metadata:
+  name: prl-cron-hearing-actual-task
+  namespace: private-law
+spec:
+  releaseName: prl-cron-hearing-actual-task
+  values:
+    job:
+      image: hmctspublic.azurecr.io/prl/cos:prod-5ffc34f-20240524155420 #{"$imagepolicy": "flux-system:demo-prl-cron-hearing-actual-task"}
```

Comment:
Hi, same here - to use this new image policy it would need included here https://github.com/hmcts/cnp-flux-config/blob/master/apps/private-law/automation/kustomization.yaml please